plugins/inputs/ravendb/ravendb.go

```diff
@@ -0,0 +1,680 @@
+package ravendb
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/influxdata/telegraf/plugins/common/tls"
```

Comment:
Please move down to the influxdata group.

Reply:
done

Reply:
Please mark as resolved.